.ci/build-linux.sh

```diff
@@ -57,9 +59,9 @@ cd ..
 
 shellcheck .ci/*.sh
 
-# If it compiled succesfully let's deploy depending on the build pipeline (Azure Pipelines).
-# Azure publishes PRs as artifacts only.
-{   [ "$IS_AZURE" = "true" ];
+# If it compiled succesfully let's deploy.
+# Azure and Cirrus publish PRs as artifacts only.
+{   [ "$IS_CI" = "true" ];
```

Comment:
Taking a look again, IS_CI doesn't really reflect what the script is asking for, a better name would probably be something like CI_HAS_ARTIFACTS.

Reply:
Done.